index.src.html

```diff
@@ -1078,6 +1078,31 @@ <h3 id="unknown-policy-values">Unknown Policy Values</h3>
 
 </section>
 
+<section>
+  <h2 id="iana">IANA Considerations</h2>
+
+  <h3 id="iana-http-headers">HTTP Headers</h3>
+
+  The HTTP Field Name Registry should be updated with the following
```

Comment:
Maybe "is to be updated" instead of "should", given the meaning of "should" per https://infra.spec.whatwg.org/#conformance.